packages/node/src/setup-config.ts

```diff
@@ -28,8 +28,8 @@ export async function setupConfig(cwd: string, opts: SetupOptions = {}) {
     }
   }
 
-  const pmResult = await getPackageManager(cwd)
-  const pm = pmResult?.name ?? 'npm'
+  const pmResult = await detect({ cwd })
+  const pm = (pmResult?.agent ?? 'npm').split('@')[0]
```

Comment:
```suggestion
  const pm = (pmResult ?? 'npm').split('@')[0]
```

I believe `detect` returns a string, so accessing `.agent` will return `undefined,` right?

Reply:
It returns an object with `agent` and `version` properties, so I believe what I have is correct

![Screenshot from 2024-08-09 12-57-23](https://github.com/user-attachments/assets/d0df089a-7151-4bed-b89c-099a4d13b834)